parasolr/query/tests/test_aliased_queryset.py

```diff
@@ -179,3 +190,33 @@ def test_get_facets(self, mock_get_facets):
         # non-aliased field is ignored
         assert 'birth' in facets['facet_ranges']
 
+    @patch('parasolr.query.queryset.SolrQuerySet.get_stats')
+    def test_get_stats(self, mock_get_stats):
+
+        sample_stats = {
+            'stats_fields': {
+                'year_i': {
+                    'min': 1918.0,
+                    'max': 1998.0
+                },
+                'start_i': {
+                    'min': 1919.0,
+                    'max': 2020.0,
+                }
+            }
+        }
+        mock_get_stats.return_value = sample_stats
+        stats = self.mysqs.get_stats()
+        # aliased field is changed to unaliased form
+        assert 'year_i' not in stats['stats_fields']
+        assert 'year' in stats['stats_fields']
+        # value of field is preserved without change
+        assert stats['stats_fields']['year'] \
+            == sample_stats['stats_fields']['year']
```

Comment:
It seems that it ought to be `sample_stats['stats_fields']['year_i']`, but this is passing and changing it makes it fail. Are we modifying the Solr return by value? Should we be?  I think I'm doing the same thing with the facets, but seeing it here gives me pause. 

I'm trying to think whether there are any implications for doing so; it might not matter in practice, but it is definitely confusing in the test. If it's only confusing here, maybe we should just use a copy for the mock return value.

Reply:
I'll mock it out to clear up confusion, and we can take some time in this or an upcoming iteration to think about how to modify (value vs reference) perhaps?